src/Altinn.App.Core/Internal/Process/ProcessTasks/Common/ProcessTaskInitializer.cs

```diff
@@ -0,0 +1,118 @@
+using Altinn.App.Core.Features;
+using Altinn.App.Core.Helpers;
+using Altinn.App.Core.Internal.App;
+using Altinn.App.Core.Internal.AppModel;
+using Altinn.App.Core.Internal.Data;
+using Altinn.App.Core.Internal.Instances;
+using Altinn.App.Core.Internal.Prefill;
+using Altinn.App.Core.Models;
+using Altinn.Platform.Storage.Interface.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.App.Core.Internal.Process.ProcessTasks;
+
+/// <inheritdoc/>
+public class ProcessTaskInitializer : IProcessTaskInitializer
+{
+    private readonly ILogger<ProcessTaskInitializer> _logger;
+    private readonly IAppMetadata _appMetadata;
+    private readonly IDataClient _dataClient;
+    private readonly IPrefill _prefillService;
+    private readonly IAppModel _appModel;
+    private readonly IInstantiationProcessor _instantiationProcessor;
+    private readonly IInstanceClient _instanceClient;
+
+    /// <inheritdoc/>
+    public ProcessTaskInitializer(ILogger<ProcessTaskInitializer> logger,
+        IAppMetadata appMetadata,
+        IDataClient dataClient,
+        IPrefill prefillService,
+        IAppModel appModel,
+        IInstantiationProcessor instantiationProcessor,
+        IInstanceClient instanceClient)
+    {
+        _logger = logger;
+        _appMetadata = appMetadata;
+        _dataClient = dataClient;
+        _prefillService = prefillService;
+        _appModel = appModel;
+        _instantiationProcessor = instantiationProcessor;
+        _instanceClient = instanceClient;
+    }
+
+    /// <inheritdoc/>
+    public async Task Initialize(string taskId, Instance instance, Dictionary<string, string> prefill)
+    {
+        _logger.LogDebug("OnStartProcessTask for {InstanceId}", instance.Id);
```

Comment:
We have lots of these issues. The problem is that if you log to standard out with a newline separating log entries, this pattern is problematic, because the user might provide two a new forged log entry between two newlines. That trick won't fool application insights, so I think we can safely ignore these.

The other issue is that C# might have a similar issue as log4j (log4shell), but after java was caught, lots of people checked dotnet for similar issues, and none was reported.